python/lsst/ts/electrometer/csc.py

```diff
@@ -154,18 +210,24 @@ async def do_setDigitalFilter(self, data):
             substates=[DetailedState.NOTREADINGSTATE],
             action="setDigitalFilter",
         )
-        await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
-        await self.controller.set_digital_filter(
-            activate_filter=data.activateFilter,
-            activate_avg_filter=data.activateAvgFilter,
-            activate_med_filter=data.activateMedFilter,
-        )
-        self.log.debug("setDigitalFilter controller interaction completed")
-        self.log.debug(f"filter_active={self.controller.filter_active}")
-        self.log.debug(f"avg_filter_active={self.controller.avg_filter_active}")
-        self.log.debug(f"median_filter_active={self.controller.median_filter_active}")
-        await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
-        self.log.info("setDigitalFilter Completed")
+        try:
+            await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
+            await self.controller.set_digital_filter(
+                activate_filter=data.activateFilter,
+                activate_avg_filter=data.activateAvgFilter,
+                activate_med_filter=data.activateMedFilter,
+            )
+            self.log.debug("setDigitalFilter controller interaction completed")
+            self.log.debug(f"filter_active={self.controller.filter_active}")
+            self.log.debug(f"avg_filter_active={self.controller.avg_filter_active}")
+            self.log.debug(
+                f"median_filter_active={self.controller.median_filter_active}"
+            )
+            await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
+            self.log.info("setDigitalFilter Completed")
+        except Exception:
+            self.log.error("Something went wrong")
```

Comment:
Same comment as above and similar comments for other new except clauses:

Please do throw away exception information unless you are sure you don't need it.

Please check ALL of your new except clauses.

Reply:
Do you mean don't throw away exception information?